react/react1/week2/react-week2/src/components/TodoTimer.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+
+export default function Timer() {
+const [seconds, setSeconds] = React.useState(1);
+
+
+ React.useEffect(() => {
+   if (seconds > 0) {
+     setTimeout(() => setSeconds(seconds + 1), 1000);
+   }
+
+   else {
+    document.write("Cannot work when timer on 0")
+   }
+
```

Comment:
A clean up function could be really useful to clear your timer when the component is destroyed. you can read about it here
[React useEffect Cleanup](https://daily-dev-tips.com/posts/react-useeffect-cleanup/)

Reply:
thanks